packages/client/src/base.ts

```diff
@@ -14,12 +14,14 @@ export abstract class MessagingChannelBase extends Emitter<{
   protected readonly http: AxiosInstance
   protected auths: { [clientId: uuid]: MessagingClientAuth } = {}
   protected headers: { [clientId: uuid]: any } = {}
+  protected adminHeader: any
```

Comment:
```suggestion
  protected adminHeader: { 'x-bp-messaging-admin-key'?: string }
```

Reply:
Looks overkill to my. It's just an object of headers, you can have anything in there. It's specified just below what we put here.